trip_reservations/src/main/kotlin/pl/edu/pg/rsww/tripreservations/TripReservationsController.kt

```diff
@@ -19,6 +24,13 @@ data class Entity(
     val data: Document,
 )
 
+object Events : Table("events") {
+    val id = long("id").autoIncrement("public.seq").uniqueIndex()
+    val entityId = uuid("entity_id")
+    val eventName = varchar("event_name", 255)
+    val data = varchar("data", 255)
```

Comment:
*If* this works then this is only a nit but you could switch this field in the schema to TEXT type so that you can avoid defining an untrue length limit in the implementation. I assume you went for this because the implementation won't ever need to store anything longer than that so that's why it's just a nit, if it does work.